homeassistant/components/here_weather/const.py

```diff
@@ -0,0 +1,393 @@
+    "dayOfWeek": {"name": "Day of Week", "unit_of_measurement": None},
+    "weekday": {"name": "Week Day", "unit_of_measurement": None},
+    "utcTime": {"name": "UTC Time", "unit_of_measurement": None},
+}
+
+DAILY_ATTRIBUTES: dict[str, dict[str, str | None]] = {
+    "daylight": {"name": "Daylight", "unit_of_measurement": None},
+    "daySegment": {"name": "Day Segment", "unit_of_measurement": None},
+    "description": {"name": "Description", "unit_of_measurement": None},
+    "skyInfo": {"name": "Sky Info", "unit_of_measurement": None},
+    "skyDescription": {"name": "Sky Description", "unit_of_measurement": None},
+    "temperature": {"name": "Temperature", "unit_of_measurement": "°C"},
+    "temperatureDesc": {"name": "Temperature Description", "unit_of_measurement": None},
+    "comfort": {"name": "Comfort", "unit_of_measurement": "°C"},
+    "humidity": {"name": "Humidity", "unit_of_measurement": "%"},
+    "dewPoint": {"name": "Dew Point", "unit_of_measurement": "°C"},
+    "precipitationProbability": {
+        "name": "Precipitation Probability",
+        "unit_of_measurement": "%",
+    },
+    "precipitationDesc": {
+        "name": "Precipitation Description",
+        "unit_of_measurement": None,
+    },
+    "rainFall": {"name": "Rain Fall", "unit_of_measurement": "cm"},
+    "snowFall": {"name": "Snow Fall", "unit_of_measurement": "cm"},
+    "airInfo": {"name": "Air Info", "unit_of_measurement": None},
+    "airDescription": {"name": "Air Description", "unit_of_measurement": None},
+    "windSpeed": {"name": "Wind Speed", "unit_of_measurement": "km/h"},
+    "windDirection": {"name": "Wind Direction", "unit_of_measurement": "°"},
+    "windDesc": {"name": "Wind Description", "unit_of_measurement": "cm"},
+    "windDescShort": {"name": "Wind Description Short", "unit_of_measurement": "cm"},
+    "beaufortScale": {"name": "Beaufort Scale", "unit_of_measurement": None},
+    "beaufortDescription": {
+        "name": "Beaufort Scale Description",
+        "unit_of_measurement": None,
+    },
+    "visibility": {"name": "Visibility", "unit_of_measurement": "km"},
+    "icon": {"name": "Icon", "unit_of_measurement": None},
+    "iconName": {"name": "Icon Name", "unit_of_measurement": None},
+    "iconLink": {"name": "Icon Link", "unit_of_measurement": None},
+    "dayOfWeek": {"name": "Day of Week", "unit_of_measurement": None},
+    "weekday": {"name": "Week Day", "unit_of_measurement": None},
+    "utcTime": {"name": "UTC Time", "unit_of_measurement": None},
+}
+
+OBSERVATION_ATTRIBUTES: dict[str, dict[str, str | None]] = {
+    "daylight": {"name": "Daylight", "unit_of_measurement": None},
+    "description": {"name": "Description", "unit_of_measurement": None},
+    "skyInfo": {"name": "Sky Info", "unit_of_measurement": None},
+    "skyDescription": {"name": "Sky Description", "unit_of_measurement": None},
+    "temperature": {"name": "Temperature", "unit_of_measurement": "°C"},
```

Comment:
Please set device class for as many measurements as possible.

If we have a common unit constant defined in Home Assistant for a certain measurement, that constant should be used when setting unit of measurement for the entity for the corresponding measurement.

https://developers.home-assistant.io/docs/core/entity/sensor#available-device-classes

Reply:
Done in https://github.com/home-assistant/core/pull/28910/commits/40949577d7e7c78c0ec625ff004b5e0d14db179f